src/Controls/src/Core/Handlers/Items/iOS/ItemsViewController.cs

```diff
@@ -200,7 +201,8 @@ void EnsureLayoutInitialized()
 
 			_initialized = true;
 
-			ItemsViewLayout.GetPrototype = GetPrototype;
+			_getPrototype ??= GetPrototype;
+			ItemsViewLayout.GetPrototype = _getPrototype;
```

Comment:
Why is this change?

Reply:
Since `GetPrototype`'s backing field is a `WeakReference<T>`, we have to save the delegate along with the `ItemsViewController` or it will get GC'd.

This way the delegate goes away when the `ItemsViewController` goes away.